### ISSUE TITLE
PHP Fatal error:  Class 'Terminus\Models\Collections\Sites' not found in PantheonAliases.php

### ISSUE THREAD

**Issue by dsnopek** [pantheon-systems/all-aliases-plugin#6]
Happens when I run `terminus sites aliases`

I'm using Terminus 0.13.3

Here's the full stack trace:

```
PHP Fatal error:  Class 'Terminus\Models\Collections\Sites' not found in /home/dsnopek/terminus/plugins/all-aliases-plugin/Commands/PantheonAliases.php on line 154
PHP Stack trace:
PHP   1. {main}() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-fs.php:0
PHP   2. include() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-fs.php:25
PHP   3. Terminus\Runner->run() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-cl.php:23
PHP   4. Terminus\Runner->runCommand() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Runner.php:212
PHP   5. Terminus\Dispatcher\Subcommand->invoke() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Runner.php:309
PHP   6. call_user_func() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/Subcommand.php:121
PHP   7. Terminus\Dispatcher\CommandFactory::Terminus\Dispatcher\{closure}() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/Subcommand.php:121
PHP   8. call_user_func() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/CommandFactory.php:134
PHP   9. Terminus\Commands\PantheonAliases->allAliases() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/CommandFactory.php:134
PHP  10. Terminus\Commands\PantheonAliases->getAliases() /home/dsnopek/terminus/plugins/all-aliases-plugin/Commands/PantheonAliases.php:71
```

**Comment by dalin-** [comment]
Same here.  I'm on 0.13.4

**Comment by kzalewski** [comment]
In your plugin, try changing the class path for the Sites class from Terminus\Models\Collections\Sites to Terminus\Collections\Sites.

It looks like they made this change at some point in the 0.13 branch.

It affected us as well.

**Comment by greg-1-anderson** [comment]
Fixed by #5
